src/routing/resources.rs

```diff
@@ -1,25 +1,24 @@
-use axum::{routing, Extension, Router};
+use axum::{
+    extract::DefaultBodyLimit,
+    routing::{post, put},
```

Comment:
I didn't found this anywhere else

Reply:
I don't understand what you're referring to, sorry

Reply:
I mean in general we include `routing` and here we have `routing::{get, post}`

Reply:
oh you want the function names to be qualified? i.e. `routing::post()` instead of `post()`?